exporter/elasticsearchexporter/internal/lru/lruset_test.go

```diff
@@ -1,10 +1,13 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+//go:build goexperiment.synctest
```

Comment:
JFYI I believe adding this Go build tag effectively skips this test in CIs, because CIs only run tests with no tags